contexts/audit/Makefile

```diff
@@ -0,0 +1,19 @@
+-include ../prerequisites/tasks.mk
+
+##>> run lint for all languages
```

Comment:
I think this needs to be updated for `audit`

Reply:
Yes, I forgot it. Done.